addons/sale_timesheet/models/project_sale_line_employee_map.py

```diff
@@ -8,17 +8,20 @@ class ProjectProductEmployeeMap(models.Model):
     _name = 'project.sale.line.employee.map'
     _description = 'Project Sales line, employee mapping'
 
+    def _get_sellable_line_domain(self, domain):
+        return self.env['project.task']._get_sellable_line_domain(domain)
+
     project_id = fields.Many2one('project.project', "Project", required=True)
     employee_id = fields.Many2one('hr.employee', "Employee", required=True)
     sale_line_id = fields.Many2one(
         'sale.order.line', "Sales Order Item",
         compute="_compute_sale_line_id", store=True, readonly=False,
-        domain="""[
+        domain=lambda self: self._get_sellable_line_domain("""[
```

Comment:
We should do the same for `so_line` field of `account.analytic.line` model in that module, no? 🧐